File: app/src/main/java/com/project/personal/app_bank/models/UserResponse.java
package com.project.personal.app_bank.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UserResponse {

    @SerializedName("userAccount")
    @Expose
    User userAccount;

    public User getUserAccount() {
        return userAccount;
    }
}

File: app/src/main/java/br/com/ibm/teste/android/services/models/UserResponse.kt
package br.com.ibm.teste.android.services.models

import com.google.gson.annotations.SerializedName

/**
 * Created by paulo.
 * Date: 10/11/18
 * Time: 17:25
 */
class UserResponse (@SerializedName("user") val user: UserAccount,
                    @SerializedName("error") val error: Error)